### PR TITLE
Chore/settings general 2 col layout

### PR DESCRIPTION
**Pull request by aiscarvalho**
The new layout in different screen widths
 
image
image
image

Other layout improvements:
- styled the "delete" button in other links the same way as the delete button in search filters
image
image
- adjusted the tab component paddings
- adjusted spacing for the horizontal rulers separating the 3 different forms
- added border and adjusted spacing in the image upload component
image

### CODE REVIEW

File: src/components/tabs/AppTabs.vue [hunk 1]
```diff
@@ -1,10 +1,16 @@
 <template>
-  <AppSelect v-model="selectedItem" :items="items" class="mb-4 md:hidden" />
-  <ul class="mb-4 hidden border-b border-primary-40 text-sm md:-mx-2 md:flex">
+  <AppSelect
+    v-model="selectedItem"
+    :items="items"
+    class="mb-4 md:mb-6 md:hidden"
```
Comment (wpf500) [inline src/components/tabs/AppTabs.vue]: I don't think you need `md:mb-6` as `md:hidden` will hide the element in any case,

File: src/pages/admin/settings/index.vue [hunk 2]
```diff
@@ -40,18 +40,18 @@ meta:
         </div>
       </AppForm>
 
-      <div class="my-6 border-b border-b-primary-40" />
+      <div class="my-8 block border-b border-b-primary-40" />
```
Comment (wpf500) [inline src/pages/admin/settings/index.vue]: I think the `<div>` will be a `block` in any case so not sure this is needed?

File: src/pages/admin/settings/index.vue [hunk 3]
```diff
@@ -84,18 +84,18 @@ meta:
           />
         </div>
       </AppForm>
-
-      <div class="my-6 border-b border-b-primary-40" />
-
+    </div>
+    <div>
+      <div class="my-8 block border-b border-b-primary-40 md:hidden" />
```
Comment (wpf500) [inline src/pages/admin/settings/index.vue]: Same as above, probably no need for `block`

File: src/components/forms/AppImageUpload.vue [hunk 4]
```diff
@@ -2,7 +2,9 @@
   <div>
     <AppLabel v-if="label" :label="label" :required="required" />
     <div class="flex items-start gap-4">
-      <div class="flex-0 basis-28 bg-grey-light">
+      <div
+        class="flex-0 basis-28 rounded border border-primary-40 bg-primary-20"
```
Comment (wpf500) [inline src/components/forms/AppImageUpload.vue]: It's really small but if you look closely you can see that the image is overlapping the rounded border. You want to add either `overflow-hidden` here or `rounded` to the `` so it respects the rounding, the former is probably preferable in this instance

**Review by wpf500 — CHANGES_REQUESTED** [review]

File: src/components/tabs/AppTabs.vue (hunk 1 above)
Comment (aiscarvalho) [reply src/components/tabs/AppTabs.vue]: Good catch; I missed that bit but it's now gone.

File: src/pages/admin/settings/index.vue (hunk 2 above)
Comment (aiscarvalho) [reply src/pages/admin/settings/index.vue]: You're correct; deleted `block`.

File: src/components/forms/AppImageUpload.vue (hunk 4 above)
Comment (aiscarvalho) [reply src/components/forms/AppImageUpload.vue]: Yes! Added `overflow-hidden`, thanks! I saw the tiny overlapping edge and forgot to fix it.

**Review by wpf500 — APPROVED** [review]

**Pull request merged**